modules/appeals_api/config/schemas/v2/200996_headers.json

```diff
@@ -17,6 +17,7 @@
         "X-VA-SSN": {
           "type": "string",
           "description": "Veteran's SSN",
+          "maxLength": 9,
```

Comment:
Since HLR v2 is in active production use we can't change schemas as readily as those we haven't released yet. Let's not worry about these maxLengths since the length is enforced by the pattern already.